packs/orion/actions/lib/node.py

```diff
@@ -0,0 +1,109 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+from utils import is_ip
+
+
+class OrionNode(object):
+    """
+    Class for holding Orion Node details.
+    """
+    def __init__(self):
+        """
+        Setup the class
+        """
+        self.__caption = None
```

Comment:
Minor style thing - "private" variables usually tend to be prefixed with a single underscore, and two underscores are used for "internal" variables.